.github/workflows/jetsocat-nuget.yml

```diff
@@ -0,0 +1,41 @@
+name: Release jetsocat nuget package
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version number (optional)'
+        required: false
+
+jobs:
+  package:
+    runs-on: [self-hosted, Linux, X64, main]
+    environment: artifactory-publish
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check out Devolutions/actions
+        uses: actions/checkout@v2
+        with:
+          repository: Devolutions/actions
+          ref: master
+          token: ${{ secrets.DEVOLUTIONSBOT_TOKEN }}
+          path: ./.github/workflows
```

Comment:
Not sure; this was added by the devops. I'll change as you suggest and we'll see when we run it :)

Reply:
There was an instance where we had to checkout the actions repo manually because of a permission issue, I don't remember exactly why. I think it should be fine in this case